Makefile

```diff
@@ -235,9 +235,8 @@ run-all: \
 .PHONY: run-all
 
 run-clickhouse: ## start clickhouse backend
-	@$(COMPOSE) up -d clickhouse
 	@echo "Waiting for clickhouse to be up and running..."
-	@$(COMPOSE_RUN) dockerize -wait tcp://clickhouse:9000 -timeout 60s
+	@$(COMPOSE) --verbose up -d --wait clickhouse
```

Comment:
Does the `--verbose` option add logs regarding the health checks?

Reply:
Nop, I forgot to remove it :grin: